Cargo.toml

```diff
@@ -68,7 +70,7 @@ serde-str = ["serde-with-str"]
 serde-with-arbitrary-precision = ["serde", "serde_json/arbitrary_precision", "serde_json/std"]
 serde-with-float = ["serde"]
 serde-with-str = ["serde"]
-std = []
+std = ["rkyv/std"]
```

Comment:
This isn't quite right, but I see what you're trying to do. Perhaps for now, we can move the `rkyv/std` feature under `rkyv` also. It just means that the feature requires the `std` feature. I'll make a code suggestion for above.

Reply:
```suggestion
std = []
```

Reply:
I think `std = ["rkyv/std"]` means if the std feature of `rust-decimal` is turned on, also turn on the `std` feature of `rkyv`?

Reply:
Unfortunately, it includes the dependency by default too. See this section: https://doc.rust-lang.org/cargo/reference/features.html#dependency-features

I see that `"rkyv?/std"` is _actually_ what we want however it requires a minimum rust version of 1.60. Unfortunately, it'll have to wait until the min version requirements catch up: https://github.com/paupino/rust-decimal#minimum-rust-compiler-version

Reply:
I see. right "rkyv?/std" is the correct one.